src/WordSearch.java

```diff
@@ -0,0 +1,29 @@
+import java.io.File;
+
+/**
+ * Getting the file path from the command prompt
+ * calling the filePath method for searching the file in system
+ */
+public class WordSearch {
+    public static void main(String[] args) throws Exception {
+        String filepath = args[0];
```

Comment:
What if no parameter is passed in by the user?

Reply:
Updated and checked the condition as you said.